docs/supported-versions.md

```diff
@@ -0,0 +1,69 @@
+# Supported GHC versions
+
+## Current GHC version support status
+
+The current support for different GHC versions is given in the following table.
+ 
+| GHC version | Last supporting HLS version                                                                                                                              | Deprecation status                       |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------- |
+| 9.2.0       | not supported yet                                                                                                                                        |                                          |
+| 9.0.1       | [current](https://github.com/haskell/haskell-language-server/releases/latest) ([partial](https://github.com/haskell/haskell-language-server/issues/297)) |                                          |
+| 8.10.7      | [current](https://github.com/haskell/haskell-language-server/releases/latest)                                                                            |                                          |
+| 8.10.6      | [current](https://github.com/haskell/haskell-language-server/releases/latest)                                                                            | will be deprecated after LTS for ghc-9.0 |
```

Comment:
Which one will come first, full 9.0.1 support or stackage LTS for 9.0? If it's the former, I think it's maybe better to mention that one instead of this, or both.

(In "both" case, we may say
> will be deprecated after complete support of GHC 9.0 as mentioned in this section

with link to the last section)

Reply:
Finally i used `will be deprecated after LTS and HLS full support for..`, hope it is complete enough

Reply:
Hmmmm, sounds like we need a term for "a fully supported major version of GHC with a LTS snapshot". That's also important for defining the major version support window. Perhaps the "mainline" GHC version? Then we could say "will be deprecated when 9.0 becomes mainline"?